### ISSUE TITLE
Potential for better tracking with a different NN framework?

### ISSUE THREAD

**Issue by John-Dean** [justinliang1020/AirPose#1]
Hi there,

Apologies, I couldn't find a contact link for you anywhere so this is the only place I could get in contact.

I see you're using MediaPipe for Pose estimation:
https://google.github.io/mediapipe/solutions/pose.html

May I suggest trying:
https://github.com/karfly/learnable-triangulation-pytorch

The main advantage of the latter is that it scales from 1 camera up to N cameras, so you could in theory not get occulusion issues regardless of however many cameras you have? I have been trying to implement this myself, but I don't have access to the Human3.6M dataset so it's proving quite a challenge!

Thanks,
John

**Comment by justinliang1020** [comment]
Hi John,

Thanks for reaching out! I agree with you that the neural net performance is somewhat lacking, especially since mediapipe is optimized to be a computer vision framework running on mobile devices/web. I mainly picked it for its ease in development.

The new neural network you sent looks interesting. I've been taking a bit of a break recently from this project due to starting work, but once I get back on it I'd be excited to rebuild it with a different ML algorithm.

Also I will add my contact info to my GitHub if you want to talk further.

Thanks,
Justin
